src/features/logging/instrument/index.js

```diff
@@ -1,13 +1,21 @@
 import { InstrumentBase } from '../../utils/instrument-base'
 import { FEATURE_NAME } from '../constants'
 import { bufferLog } from '../shared/utils'
+import { wrapLogger } from '../../../common/wrap/wrap-logger'
+import { globalScope } from '../../../common/constants/runtime'
 
 export class Instrument extends InstrumentBase {
   static featureName = FEATURE_NAME
   constructor (agentRef, auto = true) {
     super(agentRef, FEATURE_NAME, auto)
 
     const instanceEE = this.ee
+    wrapLogger(instanceEE, globalScope.console, 'log', { customAttributes: { wrappedFn: 'console.log' }, level: 'info' })
+    wrapLogger(instanceEE, globalScope.console, 'error', { customAttributes: { wrappedFn: 'console.error' }, level: 'error' })
+    wrapLogger(instanceEE, globalScope.console, 'warn', { customAttributes: { wrappedFn: 'console.warn' }, level: 'warn' })
+    wrapLogger(instanceEE, globalScope.console, 'info', { customAttributes: { wrappedFn: 'console.info' }, level: 'info' })
+    wrapLogger(instanceEE, globalScope.console, 'debug', { customAttributes: { wrappedFn: 'console.debug' }, level: 'debug' })
+    wrapLogger(instanceEE, globalScope.console, 'trace', { customAttributes: { wrappedFn: 'console.trace' }, level: 'trace' })
```

Comment:
The `wrappedFn` in postamble.js was added just for NR1.  we should discuss internally if we want to add this to all wrapped data

Reply:
@metal-messiah I did not know about that, thanks for pointing that out! 👍